book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
-  [ ] To complete what I mentioned before about your `attr_accesor`, I think you need a method to be able to add rentals to your books as well, so you may need something like `def add_rental(person, date)` and add the necessary code to the method to create the new rental.